introspection-engine/connectors/sql-introspection-connector/src/commenting_out_guardrails.rs

```diff
@@ -73,8 +75,24 @@ pub fn commenting_out_guardrails(datamodel: &mut Datamodel) -> Vec<Warning> {
         };
     }
 
-    // models without uniques / ids
+    //on postgres this is allowed, on the other dbs, this could be a symptom of missing privileges
     for model in datamodel.models_mut() {
+        if model.fields.is_empty() {
+            model.is_commented_out = true;
+            model.documentation = Some(
+                "We could not retrieve columns for the underlying table. Please check your privileges.".to_string(),
```

Comment:
I think it's fine because it's a very edgy case, but I don't know the answer: do we have a specialization mechanism so we could write another comment on postgres?

Reply:
Yeah, we could do that. Introspection has knowlegde about which db its operating